packages/playground-handbook/scripts/generateJSON.mjs

```diff
@@ -3,22 +3,21 @@
   node packages/playground-handbook/scripts/generate.mjs
 */
 
-import { writeFileSync, readdirSync, existsSync, mkdirSync, copyFileSync } from "fs"
+import { writeFileSync, readdirSync, existsSync, mkdirSync, copyFileSync } from "fs";
 
-import { dirname, join } from "path"
-import { domainToASCII, fileURLToPath } from "url"
+import { dirname, join } from "path";
+import { domainToASCII, fileURLToPath } from "url";
 
-const __dirname = dirname(fileURLToPath(import.meta.url))
-const root = join(__dirname, "..")
-const files = readdirSync(root)
-const mds = files.filter(f => f.includes(".md"))
-const types = `export type PHandbookPage = ${mds.map(t => `"${t}"`).join(" | ")}`
-writeFileSync(join(__dirname, "types.d.ts"), types)
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const root = join(__dirname, "..");
+const files = readdirSync(root);
+const mds = files.filter(f => f.includes(".md"));
+const types = `export type PHandbookPage = ${mds.map(t => `"${t}"`).join(" | ")}`;
+writeFileSync(join(__dirname, "types.d.ts"), types);
 
 /** @type {import("./types").PHandbookPage[]} */
 // prettier-ignore
 const contents = [
-  "Overview.md",            // 0
```

Comment:
Accidental auto format in this file I guess. This is the only substantive change.